iron-component-page.html

```diff
@@ -317,7 +317,7 @@
            */
           transitive: {
             type: Boolean,
-            value: false
+            value: true
```

Comment:
Unfortunately, the first two proposed solutions are both breaking changes.

I favor of the tedium solution. Only peril awaits us if we tread down the path of the breaking change.

:crystal_ball: